package.json

```diff
@@ -22,15 +22,15 @@
     "url": "http://projects.theforeman.org/projects/foreman_rh_cloud/issues"
   },
   "peerDependencies": {
-    "@theforeman/vendor": ">= 4.0.2"
+    "@theforeman/vendor": ">= 4.14.0"
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^4.0.2",
-    "@theforeman/stories": "^4.0.2",
-    "@theforeman/test": "^4.0.2",
-    "@theforeman/vendor-dev": "^4.0.2",
-    "@theforeman/eslint-plugin-foreman": "^4.0.2",
+    "@theforeman/builder": "^4.14.0",
```

Comment:
can you pin to that specific version of packaging? or using a Tilda, e.g: "~4.14.0"

Reply:
For all the packages in the file?

Reply:
Yeah if we already at it, I think it makes sense.